cmd/operator/main.go

```diff
@@ -81,11 +102,75 @@ func getIsWebhookEnabled() bool {
 	return enabled
 }
 
+// getEncoderConfig returns a concrete encoders configuration
+func getEncoderConfig(devMode bool) zapcore.EncoderConfig {
+	encoderConfig := zap.NewProductionEncoderConfig()
+	if devMode {
+		encoderConfig = zap.NewDevelopmentEncoderConfig()
+	}
+	return encoderConfig
+}
+
+// getLogWriter returns an io.writer (setting up rolling files) converted
+// into a zapcore.WriteSyncer
+func getLogWriter(path string, age, size, rotation int) zapcore.WriteSyncer {
+	lumberJackLogger := &lumberjack.Logger{
+		Filename:   path,
+		MaxSize:    size, // megabytes
+		MaxBackups: rotation,
+		MaxAge:     age, // days
+	}
+	return zapcore.AddSync(lumberJackLogger)
+}
+
+// getZapcoreLevel takes the level as string and returns the corresponding
+// zapcore.Level. If the string level is invalid, it returns the default
+// level
+func getZapcoreLevel(lvl string) zapcore.Level {
+	lvls := map[string]zapcore.Level{
```

Comment:
I think you can use `zapcore.UnmarshalText` here instead of having the map.

Reply:
Ok I will take a look, thanks.